### PR TITLE
[DOCS]: Port over "How to configure Expectation store to PostgreSQL" from RTD to Docusaurus

### PR DESCRIPTION
**Pull request by cdkini**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
- Port over .rst guide to .md
-
-

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added [unit tests](https://docs.greatexpectations.io/en/latest/contributing/testing.html#contributing-testing-writing-unit-tests) where applicable and made sure that new and existing tests are passing.
- [x] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

**Comment by cdkini** [comment]
Screenshot 2021-07-09 at 13-31-58 How to configure an Expectation store to PostgreSQL — great_expectations documentation
Screenshot 2021-07-09 at 13-31-38 How to configure an Expectation store to PostgreSQL Great Expectations

File: docs/guides/setup/configuring-metadata-stores/how-to-configure-an-expectation-store-to-postgresql.md [hunk 1]
```diff
@@ -1,5 +1,132 @@
...
+
+    ```bash
+    great_expectations --v3-api suite list
+
+    1 Expectation Suites found:
+     - exp1
+    ```
+
+
+If it would be useful to you, please comment with a +1 and feel free to add any suggestions or questions below.  Also, please reach out to us on [Slack](https://greatexpectations.io/slack) if you would like to learn more, or have any questions.
```
Comment (alexsherstinsky) [inline docs/guides/setup/configuring-metadata-stores/how-to-configure-an-expectation-store-to-postgresql.md:131]: @cdkini In my opinion, the title would read better with the following small change:
```How to configure an Expectation store to use PostgreSQL```
Thank you.

**Review by alexsherstinsky — CHANGES_REQUESTED** [review]
Small wording change requested.  The same in the other similar documents should also be updated as appropriate.

**Review by petermoyer — APPROVED** [review]

File: docs/guides/setup/configuring-metadata-stores/how-to-configure-an-expectation-store-to-postgresql.md (hunk 1 above)
Comment (cdkini) [reply docs/guides/setup/configuring-metadata-stores/how-to-configure-an-expectation-store-to-postgresql.md:131]: I agree 💯 

I've gone ahead and updated the title, the filename, and any links from within the docs (`sidebar.js` and `customize-your-deployment.md`)

**Comment by cdkini** [comment]
@alexsherstinsky I've made the name change you've requested. Do you think that change should also impact file names (i.e. `how-to-configure-an-expectation-store-to-postgresql.md` vs `how-to-configure-an-expectation-store-to-use-postgresql.md`)?

I don't think it matters since it won't impact the end user in any way. I'll go ahead and make changes to all such files in a separate PR: https://github.com/great-expectations/great_expectations/pull/3016

**Review by alexsherstinsky — APPROVED** [review]
LGTM

**Comment by alexsherstinsky** [comment]
I wonder why the checks are failing now...

**Comment by cdkini** [comment]
I'm getting a fail on `FAILED tests/integration/test_script_runner.py::test_docs[tests/integration/docusaurus/connecting_to_your_data/database/snowflake_yaml_example.py]`

I've merged changes from develop so maybe that'll fix them

**Pull request merged**